yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -595,90 +607,83 @@ Internet-Draft                 YANG Semver                 November 2021
-           rev:revision-label 3.1.0;
 
 
 
-Clarke, et al.             Expires 6 May 2022                  [Page 11]
+Clarke, et al.             Expires 8 May 2022                  [Page 11]
 
 Internet-Draft                 YANG Semver                 November 2021
 
 
-         }
-
-         revision 2017-12-31 {
-           description "Rename 'baz' to 'bar', added leaf 'wibble'";
-           rev:revision-label 3.0.0;
-           rev:non-backwards-compatible;
-         }
+    module example-versioned-module {
+      yang-version 1.1;
+      namespace "urn:example:versioned:module";
+      prefix "exvermod";
+      rev:revision-label-scheme "ysver:yang-semver";
 
-         revision 2017-10-30 {
-           description "Change the module structure";
-           rev:revision-label 2.0.0;
-           rev:non-backwards-compatible;
-         }
+      import ietf-yang-revisions { prefix "rev"; }
+      import ietf-yang-semver { prefix "ysver"; }
 
-         revision 2017-08-30 {
-           description "Clarified description of 'foo-64' leaf";
-           rev:revision-label 1.3.1;
-         }
+      description
+        "to be completed";
 
-         revision 2017-07-30 {
-           description "Added leaf foo-64";
-           rev:revision-label 1.3.0;
-         }
+      revision 2017-08-30 {
+        description "Backport 'wibble' leaf";
+        rev:revision-label 1.2.2_non_compatible;
+      }
 
-         revision 2017-04-20 {
-           description "Add new functionality, leaf 'baz'";
-           rev:revision-label 1.2.0;
-         }
+      revision 2017-07-30 {
+        description "Rename 'baz' to 'bar'";
+        rev:revision-label 1.2.1_non_compatible;
+        rev:non-backwards-compatible;
+      }
 
-         revision 2017-04-03 {
-           description "Add new functionality, leaf 'foo'";
-           rev:revision-label 1.1.0;
-         }
+      revision 2017-04-20 {
+        description "Add new functionality, leaf 'baz'";
+        rev:revision-label 1.2.0;
+      }
 
-         revision 2017-02-07 {
-           description "First release version.";
-           rev:revision-label 1.0.0;
-         }
+      revision 2017-04-03 {
+        description "Add new functionality, leaf 'foo'";
+        rev:revision-label 1.1.0;
+      }
 
-         // Note: semver rules do not apply to 0.X.Y labels.
+      revision 2017-02-07 {
+        description "First release version.";
+        rev:revision-label 1.0.0;
+      }
 
-         revision 2017-01-30 {
-           description "NBC changes to initial revision";
-           semver:module-version 0.2.0;
-         }
+      // Note: semver rules do not apply to 0.X.Y labels.
+      // The following pre-release revision statements would not
+      // appear in any final published version of a module. They
+      // are removed when the final version is published.
+      // During the pre-release phase of development, only a
+      // single one of these revision statements would appear
```

Comment:
I couldn't find any text in section 5 which states that only a single one of these revision statements would appear during pre-release.

Reply:
Section 5.1 says: Module and submodule developers SHOULD maintain only one
   revision statement in a pre-released module or submodule that
   reflects the latest revision.